### ISSUE TITLE
Problem with Timber\Image on Multisite

### ISSUE THREAD

**Issue by nexxome** [timber/timber#1312]
### Expected behavior
Image Url: http://www.example.com/wp-content/uploads/sites/2/image.jpg

### Actual behavior
Image Url: http://www.example.com/wp-content/uploads/image.jpg

### Steps to reproduce behavior
```php
switch_to_blog(1);

$image1 = new \Timber\Image( $imageFromBlog1 );
echo $image1->path();

restore_current_blog();

$image2 = new \Timber\Image( $imageFromBlog2 );
echo $image2->path();
```

### What version of WordPress, PHP and Timber are you using?
WordPress 4.7.2, PHP 7, Timber 1.2

### How did you install Timber? (for example, from GitHub, Composer/Packagist, WP.org?)
Packagist

The Problem occurs because Timber\Image stores the wp_upload_dir() in a static variable, so it doesn't 
recognize when you change the current blog:
```php
public static function wp_upload_dir() {
	static $wp_upload_dir = false;

	if ( !$wp_upload_dir ) {
		$wp_upload_dir = wp_upload_dir();
	}

	return $wp_upload_dir;
}
```

**Comment by gchtr** [comment]
I ran into the same issue and submitted a pull request to fix this in #2478.

**Comment by gchtr** [comment]
I guess I can close this. The fix is in the [1.19.1 release](https://github.com/timber/timber/releases/tag/1.19.1).
